src/ags799/bootlaces.clj

```diff
@@ -2,17 +2,11 @@
   {:boot/export-tasks true}
   (:gen-class)
   (:require [boot.core :as boot]
+            [boot.task.built-in :refer [aot pom uber jar install]]
             [clojure.java.shell :refer [sh]]
             [tolitius.boot-check :as check]))
 
-(defn version
-  "Returns the project's version.
-
-  The version is the HEAD commit's short hash. This is done to support
-  continuous deployment: if we deploy every commit on the deployed branch,
-  then we won't necessarily tag every commit that is deployed. However, those
-  commits will have a hash. That's what we use as our version."
-  []
+(defn- short-commit-hash []
```

Comment:
surprisingly, i couldn't put this private function at the bottom, since it seems you can only refer to functions with an earlier definition